ui/controls/cascading-list.reel/cascading-list-item.reel/cascading-list-item.html

```diff
@@ -24,6 +31,16 @@
                 "value": {"<-": "@owner.isCollection ? path(@owner.userInterfaceDescriptor.collectionNameExpression) : @owner.object.path(@owner.userInterfaceDescriptor.nameExpression)"}
             }
         },
+        "helpButton": {
+            "prototype": "blue-shark/ui/button.reel",
+            "properties": {
+                "element": {"#": "helpButton"},
+                "value": "Help"
+            },
+            "bindings": {
+                "classList.has('is-visible')": {"<-": "@owner.object.path(@owner.userInterfaceDescriptor.helpMessage)"}
```

Comment:
Maybe you could simplify this frb expression by checking helpMessage.value is defined.